Gruntfile.js

```diff
@@ -8,7 +8,7 @@ module.exports = function (grunt) {
         // Configurable paths
         config: {
             lintFiles: [
-                '**/*.js'
+                'angular-bind-html-compile.js'
```

Comment:
Could you change this to  
```
[
    '**/*.js',
    '!*.min.js'
]
```
So we lint the Gruntfile too and any future potential files?